molo/profiles/admin.py

```diff
@@ -129,6 +129,21 @@ def get_queryset(self, request):
         return User.objects.filter(profile__site=request.site)
 
 
+class UserProfileModelAdmin(WagtailModelAdmin, ProfileUserAdmin):
+    model = UserProfile
+    menu_label = 'Permissions'
```

Comment:
@codiebeulaine please update the label here

Reply:
@saeedmb8 what was i supposed to change it to again?